src/main/java/kitchenpos/order/application/OrderService.java

```diff
@@ -18,24 +22,30 @@
 @Service
 @Transactional
 public class OrderService {
-    private final MenuRepository menuRepository;
     private final OrderRepository orderRepository;
     private final OrderTableRepository orderTableRepository;
+    private final ApplicationEventPublisher publisher;
 
     public OrderService(
-            MenuRepository menuRepository, OrderRepository orderRepository,
-            OrderTableRepository orderTableRepository) {
-        this.menuRepository = menuRepository;
+            OrderRepository orderRepository,
+            OrderTableRepository orderTableRepository, ApplicationEventPublisher publisher) {
         this.orderRepository = orderRepository;
         this.orderTableRepository = orderTableRepository;
+        this.publisher = publisher;
     }
 
     public OrderResponse create(OrderCreateRequest orderCreateRequest) {
-        OrderLineItems orderLineItems = generateOrderLineItems(orderCreateRequest);
-
         OrderTable orderTable = generateOrderTable(orderCreateRequest);
 
-        Order savedOrder = generateOrder(orderTable, orderLineItems);
+        Order savedOrder = orderRepository.save(
+                new Order(
+                        orderTable,
+                        OrderStatus.COOKING.name(),
```

Comment:
이 부분은 `OrderStatus.COOKING`을 바로 사용하시는건 어떨까요?? 
`Enum`의 `name()`으로 되어있으면 조건 비교 할 때 매번 `name()`으로 해야 합니다 😄 

```suggestion
                        OrderStatus.COOKING,
```
이 부분 개선 검토 부탁 드립니다 😄

Reply:
수정하였습니다 :)